api/wb/serializers.py

```diff
@@ -3,19 +3,22 @@
 from rest_framework import serializers as ser
 from rest_framework import exceptions
 
+from api.base import utils
+
 from website.files import exceptions as file_exceptions
-from api.base.serializers import IDField, ShowIfVersion
+from api.base.serializers import IDField
 
 
 class DestinationSerializer(ser.Serializer):
     parent = ser.CharField(write_only=True)
     target = ser.CharField(write_only=True)
     name = ser.CharField(write_only=True, allow_blank=True, allow_null=True)
-    node = ShowIfVersion(
```

Comment:
As I noted in the PR description `ShowIfVersion` doesn't actually effect write only fields. I tried mightily to make a RequireIfVersion` field, but since in the field model we are working outside the request context this was difficult to do.